<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单自由度模拟振动台控制系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rightFrame">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>0</y>
      <width>235</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/Icon/Icon/startIcon.png</normaloff>:/Icon/Icon/startIcon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/Icon/Icon/stopIcon.png</normaloff>:/Icon/Icon/stopIcon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labeStatusl">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>试验状态</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="lastFrame">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>270</y>
      <width>235</width>
      <height>252</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="labeStatusl_2">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>试验信息</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>新建试验</string>
     </property>
     <addaction name="action_SineWaveTest"/>
     <addaction name="action_EarthWaveTest"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="action_ToolFlag"/>
    <addaction name="action_StatusFlag"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_ChannelParameters"/>
    <addaction name="action_ControlParameters"/>
    <addaction name="action_LimitedParameters"/>
   </widget>
   <widget class="QMenu" name="menu_Command">
    <property name="title">
     <string>命令(&amp;C)</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Pause"/>
    <addaction name="action_Continue"/>
    <addaction name="action_Stop"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>数据(&amp;D)</string>
    </property>
    <addaction name="action_Pointer"/>
    <addaction name="action_ZoomIn"/>
    <addaction name="action_ZoomOut"/>
    <addaction name="separator"/>
    <addaction name="action_StorageSettings"/>
    <addaction name="action_SaveAsPicture"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_HelpF1"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Command"/>
   <addaction name="menu_Data"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Pointer"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/Icon/fileopen.png</normaloff>:/Icon/Icon/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开试验</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/Icon/save.png</normaloff>:/Icon/Icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存试验</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>退出系统</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_SineWaveTest">
   <property name="text">
    <string>正弦波试验</string>
   </property>
  </action>
  <action name="action_EarthWaveTest">
   <property name="text">
    <string>地震波试验</string>
   </property>
  </action>
  <action name="action_ToolFlag">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
  </action>
  <action name="action_StatusFlag">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="action_HelpF1">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="action_Continue">
   <property name="text">
    <string>继续</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_StorageSettings">
   <property name="text">
    <string>保存设置</string>
   </property>
  </action>
  <action name="action_SaveAsPicture">
   <property name="text">
    <string>保存为图片</string>
   </property>
  </action>
  <action name="action_ChannelParameters">
   <property name="text">
    <string>通道参数</string>
   </property>
  </action>
  <action name="action_ControlParameters">
   <property name="text">
    <string>控制参数</string>
   </property>
  </action>
  <action name="action_LimitedParameters">
   <property name="text">
    <string>限制参数</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/Icon/zoomin.png</normaloff>:/Icon/Icon/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/Icon/zoomout.png</normaloff>:/Icon/Icon/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_Pointer">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icon/Icon/pointer.png</normaloff>:/Icon/Icon/pointer.png</iconset>
   </property>
   <property name="text">
    <string>指针</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
